tests/test_bump.py

```diff
@@ -1,36 +1,56 @@
-"""
-Tests related to bump.py module.
-"""
+"""Tests related to bump.py module."""
 
 import numpy as np
-import os
 import pytest
 
 from pandas import read_csv
 
-from resspect import bump, fit_bump
+from resspect.bump import bump, fit_bump, protected_exponent, protected_sig
 
 
-def test_bump():
-    """
-    Test the Bump function evaluation.
-    """
+def test_protected_exponent():
+    """Test the protected_exponent() function."""
+    values = np.arange(0, 20)
+    results = protected_exponent(values)
+
+    # Input values 0-10 should all return exp(x). Anything above that should
+    # return exp(10.0).
+    np.testing.assert_allclose(results[:11], np.exp(np.arange(0, 11)))
+    np.testing.assert_allclose(results[10:], [np.exp(10.0)] * 10)
+
+
+def test_protected_sig():
+    """Test the protected_sig() function."""
+    values = np.arange(-20, 10)
+    results = protected_sig(values)
 
+    expected_upper = 1.0 / (1.0 + np.exp(np.arange(10, -10, -1)))
+    expected_lower = 1.0 / (1.0 + np.exp(np.full(10, 10.0)))
 
-    time = np.array([0])
+    # Input values [-20, -10] should return 1.0 / (1.0 + exp(10.0))
+    # and input values [-10, 10] should return 1.0 / (1.0 + exp(-x))
+    np.testing.assert_allclose(results[:10], expected_lower)
+    np.testing.assert_allclose(results[10:], expected_upper)
+
+
+def test_bump():
+    """Test the Bump function evaluation."""
```

Comment:
No obligation to do anything, just an observation - This seems like it would make a nice doctest. I'm not sure that we use that in any of our packages currently but it seems like this would be a good example if we wanted to start.